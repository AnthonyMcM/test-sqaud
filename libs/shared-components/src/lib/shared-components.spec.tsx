import { render } from '@testing-library/react';

import MainHeading from './shared-components';

describe('SharedComponents', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<MainHeading />);
    expect(baseElement).toMatchSnapshot();
  });

  it('Should have text', () => {
    const result = render(<MainHeading children="Test" />);
    result.getByText('Test');
  });
});
