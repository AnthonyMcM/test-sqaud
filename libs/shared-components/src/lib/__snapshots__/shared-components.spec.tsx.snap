// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharedComponents should render successfully 1`] = `
<body>
  <div>
    <div
      class="shared-components__StyledMainHeading-d7vfww-0 eTsNWm"
    >
      <h1 />
    </div>
  </div>
</body>
`;
